lib/views/BinnedMeanGraph.js

```diff
@@ -105,16 +107,16 @@ class BinnedMeanGraph extends Graph {
                 .attr('class', 'binned_mean_guide binned_mean_x_guide');
 
             super.drawXLineGuide(xGuide, xG, chartOpts, {
-                xProp: 'x_mean',
-                yProp: 'y_mean',
+                xProp          : 'x_mean',
+                yProp          : 'y_mean',
```

Comment:
Haha this diff is truly a battle between OCD formatting styles. I like it :smile: